package-configuration/linux-cli/test_cli_install.py

```diff
@@ -51,10 +51,14 @@ def test_full_cli_install(self):
         finally:
             shutil.rmtree(tempdir)
 
-    def test_install_from_source(self):
+    def test_install_from_source_with_requirements(self):
         tempdir = tempfile.mkdtemp()
+        temp_requirements_file = tempfile.NamedTemporaryFile(delete=True)
+        with open(temp_requirements_file.name, 'w') as requirements_file:
```

Comment:
oh, in that case there's two small issues:
* `temp_requirements_file` keeps file handler open
* you delete the entire tmp folder which might fail (ignore_errors=True?) and generally not nice :cry: